mmds/src/data_store.rs

```diff
@@ -216,28 +258,53 @@ mod tests {
     }
 
     #[test]
-    fn test_invalid_types() {
+    fn test_update_data_store() {
         let mut mmds = Mmds::default();
+
         let data = r#"{
             "name": {
                 "first": "John",
                 "second": "Doe"
             },
-            "age": 43
+            "age": "43"
         }"#;
+        let data_store: Value = serde_json::from_str(data).unwrap();
+        assert_eq!(mmds.put_data(data_store), Ok(()));
```

Comment:
nit: ` assert!(mmds.put_data(data_store).is_ok())`
Same for the other asserts that check the result is ok.